cmd/kubeadm/app/util/runtime/runtime_test.go

```diff
@@ -128,7 +128,7 @@ func TestIsRunning(t *testing.T) {
 func TestListKubeContainers(t *testing.T) {
 	fcmd := fakeexec.FakeCmd{
 		CombinedOutputScript: []fakeexec.FakeCombinedOutputAction{
-			func() ([]byte, error) { return []byte("k8s_p1\nk8s_p2\nid3"), nil },
+			func() ([]byte, error) { return []byte("k8s_p1\nk8s_p2"), nil },
```

Comment:
Add reasons to change this test?

Reply:
IIUC, this test does not need to be changed, @kad right?

Reply:
@dixudx that test expects 2 items in the list. As we removed the check number of items becomes 3, so one needs to be removed.